### PR TITLE
[12.0][MIG] account_voucher

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Migration scripts for `account_voucher` module.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please do rebase

**Comment by MiquelRForgeFlow** [comment]
Done

File: addons/account_voucher/migrations/12.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---Fields in module 'account_voucher'---
+account_voucher / account.voucher          / company_id (many2one)         : is now stored
+# DONE: post-migration: filled from account_journal
+
+account_voucher / account.voucher          / currency_id (many2one)        : is now stored
+# DONE: post-migration: compute the field
```
Comment (pedrobaeza) [inline addons/account_voucher/migrations/12.0.1.0/openupgrade_analysis_work.txt]: No need of computing this again. When the field is not stored and gets stored, the compute is done on regular update process.

File: addons/account_voucher/migrations/12.0.1.0/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---Fields in module 'account_voucher'---
+account_voucher / account.voucher          / company_id (many2one)         : is now stored
+# DONE: post-migration: filled from account_journal
```
Comment (pedrobaeza) [inline addons/account_voucher/migrations/12.0.1.0/openupgrade_analysis_work.txt]: No need of getting this again. When the field is not stored and gets stored, the compute is done on regular update process.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: addons/account_voucher/migrations/12.0.1.0/openupgrade_analysis_work.txt (hunk 2 above)
Comment (MiquelRForgeFlow) [reply addons/account_voucher/migrations/12.0.1.0/openupgrade_analysis_work.txt]: company_id is not a compute, is a related field.

Comment (pedrobaeza) [reply addons/account_voucher/migrations/12.0.1.0/openupgrade_analysis_work.txt]: It applies the same. And a computing is done also on related fields :wink:

**Comment by MiquelRForgeFlow** [comment]
Done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**